lib/info.js

```diff
@@ -108,10 +111,32 @@ module.exports = function getInfo(link, options, callback) {
     } else {
       dashmpd2 = results[1].dashmpd;
     }
+    config.args.relatedIds = relatedIds;
     gotConfig(options, description, config, dashmpd2, callback);
   });
 };
 
+/**
+ * @param {String} HTML body
```

Comment:
Just "body" here, not "HTML body".

Description of a parameter comes after the parameter name.